lib/livebook/runtime/evaluator/io_proxy.ex

```diff
@@ -245,6 +245,12 @@ defmodule Livebook.Runtime.Evaluator.IOProxy do
     {:ok, state}
   end
 
+  # defp io_request({:livebook_put_doctest, output}, state) do
+  #   state = flush_buffer(state)
+  #   send(state.send_to, {:runtime_doctest_output, state.ref, output})
+  #   {:ok, state}
+  # end
+
```

Comment:
not sure why/what is the function of `io_proxy.ex` but I am following the same pattern for delivering messages as `:livebook_put_output`, which goes through this module